codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/customizations/SmithyTypesPubUseExtra.kt

```diff
@@ -79,4 +80,16 @@ fun pubUseSmithyPrimitives(codegenContext: CodegenContext, model: Model, rustCra
             "SdkBody" to RuntimeType.smithyTypes(rc).resolve("body::SdkBody"),
         )
     }
+    if (codegenContext.serviceShape.hasEventStreamOperations(model)) {
```

Comment:
I think these should get their own `crate::primitives::event_stream` module to group them together.

Reply:
Groped them together in c8c293b.